### DISCUSSION TITLE
How to add externally_connectable?, since it's only in chrome, and give the error

### DISCUSSION THREAD

**Discussion by MoriorInvictus** [antfu/vitesse-webext#98] [Q&A]
I need to add externally_connectable to my extension manifest, but all they give me an error, that it's not described in webextension-polyfill, how to disable that? what if I make an extension specifically for chromium since it's just a joke to create an extension that I'm making, make it for firefox, this extension is for the game, and playing any game in something except chromium is shooting into yourself.

`
	externally_connectable: {
		matches: ["*://*.lol.com/*"]
	}
`

**Comment by seomikewaltman** [comment]
I just extended the manifest interface like this on `manifest.ts`

```
interface WebExtensionExternal extends Manifest.WebExtensionManifest {
    externally_connectable: {
        matches: string[]
    }
    key?: string
}
```
